src/event.js

```diff
@@ -0,0 +1,129 @@
+class Events {
+	constructor() {
+		this.eventsBox = document.getElementById('events');
+		this.eventName = document.querySelector('.eventName');
+		this.eventDate = document.querySelector('.eventDate');
+		this.eventImg = document.querySelector('.eventImg');
+		this.buttonLeft = document.querySelector(' .button-left');
+		this.buttonRight = document.querySelector(' .button-right');
+		this.counter = 0;
+		this.events = [];
+		this.mobile = window.matchMedia('min-width:576px');
```

Comment:
```suggestion
		this.mobile = window.matchMedia('(max-width:576px)');
```
całe wyrażenie musi być w nawiasach, jeżeli to jest mobilna apka to chyba chodziło o max width